### PR TITLE
DOCS-1364: Document creating datasets

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: assets/manage/ml/train-model-menu.png
Comment (agreenb) [inline assets/manage/ml/train-model-menu.png:1]: [supernit] this screenshot needs the `datasets` flag added to the URL to have `DATASETS` showing up at the top banner.

File: docs/manage/data/_index.md
Comment (agreenb) [inline docs/manage/data/_index.md:11]: Can update SME to me

File: docs/manage/data/dataset.md
Comment (agreenb) [inline docs/manage/data/dataset.md:11]: Can make this Tahiya + me

File: docs/manage/data/dataset.md
Comment (agreenb) [inline docs/manage/data/dataset.md:86]: We can add it in explicitly when https://viam.atlassian.net/browse/DATA-2166 is resolved, but might be worth mentioning that it can be created directly on the image, the same way that you add data to a dataset below.

Though I know it's already written below that `You can select an existing dataset or create a new one from this dropdown` so perhaps too redundant?

File: docs/manage/data/dataset.md
Comment (agreenb) [inline docs/manage/data/dataset.md:98]: [supernit] an image

File: docs/manage/data/dataset.md
Comment (agreenb) [inline docs/manage/data/dataset.md:109]: You can also view the data by clicking on the dataset directly from the image sidebar, since each listed dataset on the image is a direct link into that dataset page.

and [nit], perhaps 'by clicking on your dataset name'?

File: docs/manage/data/dataset.md
Comment (agreenb) [inline docs/manage/data/dataset.md:111]: This took me a few reads — perhaps:

`
You can click on any image in the dataset and use its appearing menu in order to add or remove bounding box labels or tags, or remove that image from the dataset
`

File: docs/manage/data/dataset.md
Comment (agreenb) [inline docs/manage/data/dataset.md:119]: To make it clear that this is on the listed dataset page, perhaps 'right click on the dataset in https://app.viam.com/data/datasets'?

File: docs/manage/ml/train-model.md
Comment (agreenb) [inline docs/manage/ml/train-model.md:10]: Can update this to Tahiya + me

File: docs/manage/ml/train-model.md
Comment (agreenb) [inline docs/manage/ml/train-model.md:13]: Labels are comprised of image classification tags and bounding boxes, so would either have this be 'you can label images collected' or 'you can add classification tags or bounding boxes to images collected'

File: docs/manage/ml/train-model.md
Comment (agreenb) [inline docs/manage/ml/train-model.md:35]: In the section below:

> Once the model has finished training, it becomes visible in the **Models** section of the page.

A notable addition is that the model will have a link to the dataset it was trained on. You can see this in https://app.viam.dev/data/models if you've trained a model on a dataset, e.g.:
image

It's worth noting that the dataset can have images added or removed to it since that model was trained, and it's not currently versioned.

**Review by agreenb — CHANGES_REQUESTED** [review]
Looks fantastic, thanks so much for the lightning fast turnaround time! First pass, but already looks close to ready.

File: docs/manage/data/_index.md
Comment (npentrel) [inline docs/manage/data/_index.md:11]:
```suggestion
# SME: Alexa Greenberg
```

File: docs/manage/data/dataset.md
Comment (npentrel) [inline docs/manage/data/dataset.md:11]:
```suggestion
# SME: Tahiya Salam and Alexa Greenberg
```

File: docs/manage/data/dataset.md
Comment (npentrel) [inline docs/manage/data/dataset.md:98]:
```suggestion
To add an image to a dataset, click on the image and select the dataset in the Datasets dropdown.
```

File: docs/manage/data/dataset.md
Comment (npentrel) [reply docs/manage/data/dataset.md:111]: sounds good thx!

Comment (npentrel) [inline docs/manage/data/dataset.md:111]:
```suggestion
You can click on any image in the dataset and use its appearing menu in order to add or remove bounding box labels or tags, or remove that image from the dataset.
```

File: docs/manage/data/dataset.md
Comment (npentrel) [inline docs/manage/data/dataset.md:119]:
```suggestion
To delete a dataset, go to the [**DATASETS** subtab](https://app.viam.com/data/datasets), right click on the dataset and click on the **Delete** button that appears.
```

File: docs/manage/data/dataset.md
Comment (npentrel) [inline docs/manage/data/dataset.md:86]:
```suggestion
On the [**DATASETS** subtab](https://app.viam.com/data/datasets), click **Create dataset**, enter a name for your dataset and click **Create dataset** again.
```

File: docs/manage/data/dataset.md
Comment (npentrel) [inline docs/manage/data/dataset.md:109]:
```suggestion
Once you have added data to your dataset, you can view the data from the [**DATASETS** subtab](https://app.viam.com/data/datasets) by clicking on your dataset.
```

File: docs/manage/ml/train-model.md
Comment (npentrel) [inline docs/manage/ml/train-model.md:10]:
```suggestion
# SME: Tahiya + Alexa Greenberg
```

File: docs/manage/ml/train-model.md
Comment (npentrel) [reply docs/manage/ml/train-model.md:13]:
```suggestion
You can add classification tags or bounding boxes to [images collected](/services/data/configure-data-capture/) by robots, add them to a dataset, and use the annotated data to train a **Single Label Classification Model**, **Multi Label Classification Model** or **Object Detection Model** within Viam.
```

File: docs/manage/data/dataset.md
Comment (npentrel) [inline docs/manage/data/dataset.md:90]:
```suggestion
```

File: docs/manage/ml/train-model.md
Comment (npentrel) [reply docs/manage/ml/train-model.md:35]: I can't confirm that the Dataset is visible Alexa. Is this still in progress?
Screenshot 2023-11-21 at 10 42 51

Comment (agreenb) [reply docs/manage/ml/train-model.md:39]: This will be fixed now if you retrain a new model on a dataset.

File: assets/tutorials/pet-treat-dispenser/app-data-images.png
Comment (agreenb) [inline assets/tutorials/pet-treat-dispenser/app-data-images.png:1]: Super subtle difference, but after https://github.com/viamrobotics/app/pull/2984 is merged (where we get rid of the old flow and replace with the new dataasets fow), we will no longer show 'Train model' on the main Images tab, since we'll only allow you to do so from the Datasets tab. This'll be reflected in app.viam.dev in the next hour or so, and in app.viam.com tomorrow at the 11am prod push.

This is pretty nit so these screenshots can always be updated after the prod update.

**Review by agreenb — APPROVED** [review]
LGTM, thanks so much for iterating, updating the tutorials, giving feedback on the flow, and helping us file and fix some bugs! This was a really fantastic collaboration and excited for this feature to be so well documented.

File: docs/manage/ml/train-model.md
Comment (agreenb) [reply docs/manage/ml/train-model.md:39]: This would be the only blocking update before merging these docs; I think my other one regarding the subtle difference with 'Train' in the screenshots can be updated after.

**Comment by agreenb** [comment]
Last update that @sbal13 just fit in before today's app deployment (fantastic work!)

You can right-click on the List Datasets view in order to delete or rename the dataset. 
image

You can now also rename/delete on the actual Dataset page itself.

I'll update here in ~20 minutes when this is available in staging and provide some screenshots, since the deployment link was torn down while it's being merged.

**Comment by agreenb** [comment]
This is now available in staging!

Example of the rename/delete on the dataset page itself:

https://github.com/viamrobotics/docs/assets/3794748/09ba63b1-3a81-45d4-9bbd-af53f8d4b136

**Comment by agreenb** [comment]
Confirming that all is available in prod. No need to block on my re-review of the newer screenshots; all will LGTM after they're updated.

**Pull request merged**
